.tree-sitter-lint/tree-sitter-lint-local/Cargo.toml

```diff
@@ -7,3 +7,6 @@ edition = "2021"
 tree-sitter-lint = { path = "../.." }
 local_rules = { path = "../local_rules" }
 tree-sitter-lint-plugin-replace-foo-with = { path = "../.././tests/fixtures/tree-sitter-lint-plugin-replace-foo-with" }
+
+[[bin]]
+name = "tree-sitter-lint-local"
```

Comment:
Now the custom local "binary" is used as both an executable (by the `tree-sitter-lint` command-line usage) and as a crate (by `tree-sitter-lint-lsp`, currently a hard-coded single version of that but what would presumably be a custom local binary version of that)

All of these changes are just made manually at the moment (will be a separate task to update the "auto-generation" code to now spit this out instead)